sklearn/svm/src/libsvm/LIBSVM_CHANGES

```diff
@@ -4,5 +4,6 @@ This is here mainly as checklist for incorporation of new versions of libsvm.
 
   * Add copyright to files svm.cpp and svm.h
   * Add random_seed support and call to srand in fit function
+  * Improved random number generator (fix on windows)
```

Comment:
It'd be nice to have the PR# as a reference in these inline comments for future reference.

Reply:
ok, I'll insert <https://github.com/scikit-learn/scikit-learn/pull/13511#issuecomment-481729756> as for the others